e2e/_suites/fleet/stand-alone.go

```diff
@@ -85,6 +85,12 @@ func (fts *FleetTestSuite) startStandAloneAgent(image string, flavour string, en
 
 	dockerImageTag := common.BeatVersion
 
+	common.ProfileEnv["elasticAgentDockerNamespace"] = utils.GetDockerNamespaceEnvVar("beats")
```

Comment:
Out of curiosity, I see `common.ProfileEnv["elasticAgentDockerNamespace"] = utils.GetDockerNamespaceEnvVar("beats")` but the docker namespace is hardcoded to `beats` or `obs-ci` in https://github.com/elastic/e2e-testing/pull/1498/files#diff-e63e468667c98b8b0f8e6f12019ebd01f3c32b60cbcfe1d768b533a2d7572af5R394

Should the docker registry be customised too?

Reply:
We only use the Load & Tag docker images when we download a Docker image in TAR format from a GCP bucket. And when loading that binary into the docker engine, it always uses the "official"/original docker namespace: beats. That's why we load it into "beats" and tag it into "obs-ci". And that only happens for GCP snapshots, because for Beats merges, we already pushed the images to the `obs-ci` namespace.